tests/auth_helper.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from hushline.model import User
+
+
+def register_user(client, username, password):
+    # Prepare the environment to not require invite codes
+    os.environ["REGISTRATION_CODES_REQUIRED"] = "False"
```

Comment:
This should be set using an `app` or `config` fixture as this is probably the default for all tests

Reply:
👍